src/components/header.js

```diff
@@ -1,6 +1,275 @@
-import React from "react";
+import { HamburgerIcon, CloseIcon, ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  Collapse,
+  Icon,
+  IconButton,
+  Link,
+  Stack,
+  useDisclosure,
+} from "@chakra-ui/react";
+import {
+  Text,
+  Box,
+  Image,
+  Container,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Button,
+} from "@chakra-ui/react";
+import { Search } from "./search";
 
+const navItems = [
+  {
+    label: "Movies",
+    children: [
+      {
+        label: "Popular",
+        href: "#",
+      },
+      {
+        label: "Now Playing",
+        href: "#",
+      },
+      {
+        label: "Upcoming",
+        href: "#",
+      },
+      {
+        label: "Top Rated",
+        href: "#",
+      },
+    ],
+  },
+  {
+    label: "Tv Shows",
+    children: [
+      {
+        label: "Popular",
+        href: "#",
+      },
+      {
+        label: "Airing Today",
+        href: "#",
+      },
+      {
+        label: "On Tv",
+        href: "#",
+      },
+      {
+        label: "Top Rated",
+        href: "#",
+      },
+    ],
+  },
+  {
+    label: "People",
+    children: [
+      {
+        label: "Popular People",
+        href: "#",
+      },
+    ],
+  },
+
+  {
+    label: "About Us",
+    children: [
+      {
+        label: "About Us",
+        href: "#",
+      },
+      {
+        label: "Contact Us",
+        href: "#",
+      },
+    ],
+  },
+];
 
 export function Header() {
-    return <h1> This is Header</h1>;
-}
\ No newline at end of file
+  const { isOpen, onToggle } = useDisclosure();
+
+  return (
+    <Box
+      bg="white"
+      zIndex="popover"
+      sx={{
+        position: "sticky",
+        top: "0",
+      }}
+    >
+      <Flex minH={"80px"} boxShadow="lg" align={"center"}>
+        <Container maxW="container.xl" display={"flex"}>
+          <Flex
+            alignItems="center"
+            flex={{ base: 1 }}
+            justify={{ base: "center", md: "start" }}
+          >
+            <Flex alignItems="center">
+              <Image h="50px" src="https://i.postimg.cc/rm8LXXgw/Asset-1.png" />
+              <Text fontWeight="bold" fontSize="2xl" pl={4} mt="-10px">
+                movieFreak
+              </Text>
+            </Flex>
+            <Flex
+              alignItems="center"
+              display={{ base: "none", md: "flex" }}
+              ml={10}
+            >
+              <Navigation />
+              <Box
+                position="relative"
+                pl={20}
+                display={{ sm: "none", md: "none", lg: "none", xl: "flex" }}
+              >
+                <Search />
+              </Box>
+            </Flex>
+            <Flex
+              pl={6}
+              alignItems="center"
+              flex={{ base: 1 }}
+              justify={"right"}
```

Comment:
My personal preference for writing props that are strings is to omit the curly brackets, there's no difference in how it will be read though. This is fine as well, but having a string is more HTML-like in my opinion. e.g. `justify="right"` instead of `justify={"right"}`.